src/instance/minecraft/chat/UnmuteChat.ts

```diff
@@ -5,11 +5,12 @@ import { EventType } from '../../../common/ApplicationEvent'
 
 export default {
   onChat: function (context: MinecraftChatContext): void {
-    const regex = /^(?:\[[+A-Z]{1,10}] ){0,3}\w{3,32} has unmuted (?:\[[+A-Z]{1,10}] ){0,3}(\w{3,32})/g
+    const regex =
+      /^(?:\[[+A-Z]{1,10}] ){0,3}(\w{3,32}) has unmuted (the guild chat|(?:\[[+A-Z]{1,10}] ){0,3}(\w{3,32}))/g
 
     const match = regex.exec(context.message)
     if (match != undefined) {
-      const username = match[1]
+      const username = match[1] === 'the' ? match[0] : match[1]
```

Comment:
You already changed regex to detect "the guild chat". Double changing is too complex.